R/filter_occs_time.R

```diff
@@ -0,0 +1,277 @@
+#' filter_occs_time
```

Comment:
Sorry, I wasn't saying you needed to change the name of the function (in fact, I think the old function name worked quite well and matches our existing naming scheme), just that this first line should literally be a title for the function (see suggestion)

```suggestion
#' Filter fossil occurrences by time
```